content/usando-virtualenv-com-virtualenvwrapper.md

```diff
@@ -0,0 +1,61 @@
+Title: Usando virtualenv com virtualenvwrapper
+Slug: usando-virtualenv-com-virtualenvwrapper
+Date: 2015-05-14 00:11
+Tags: python,desenvolvimento
+Author: André Luiz
+Email:  contato@xdvl.info
+Github: dvl
+Site: http://dvl.rocks/
+Twitter: xdvl
+Category: Python
+
+O uso do `virtualenv` é um item quase que obrigatório em qualquer ambiente de desenvolvimento Python, é usado para isolar os pacotes do projeto dos pacotes do sistema.
+
+O que vejo poucas pessoas usando é o `virtualenvwrapper` que visa a resolver vários problemas criados pelo `virtualenv` como a necessidade de se usar sempre `source bin/activate` e os arquivos do `virtualenv` sendo colocados lado-a-lado com os arquivos do seu projeto.
+
+## Instalação
+
+Primeiro vamos a instalação dos pacotes necessários (essa possivelmente deverá ser a ultima vez que você instala algo globalmente no seu sistema):
+
+    $ sudo pip install virtualenv virtualenvwrapper
```

Comment:
Acho melhor partir da premissa que a pessoa não sabe o que é pip e virtualenv, porque quem já sabe, vai ver logo de cara o que ela pode ignorar, e vai usar só o que for necessario. O novato não vai saber disso. Então eu acho melhor explicar tudo do inicio.